http-gateway/web/README.md

```diff
@@ -5,6 +5,7 @@
     nodejs
 
 ## Configuration
+
```

Comment:
Please move the UI documentation to: `https://github.com/plgd-dev/cloud/tree/v2/docs/guide/developing`. Create new file `dashboard.md`.

Group `### Requirements`, `### Configuration`, `### Installation`,  `### Starting the development server` and `### Building` under `Building` and use code formatting for the commands. 

Translation is a separate group `## Translation`, same as `## Branding`.

Reply:
Then append here a new entry: https://github.com/plgd-dev/cloud/blob/v2/docs/.vuepress/config.js#L48

`developing/dashboard`

You can test it by running `npm docs:dev` in the root folder -> `/plgd-dev/cloud/.`

Reply:
Applied in https://github.com/plgd-dev/cloud/pull/174